src/Components/AccountPageSubPages/AvatarSelectionElement.jsx

```diff
@@ -0,0 +1,40 @@
+import { React } from "react";
+import { ImageListItem, Tooltip, IconButton } from "@mui/material";
+import { changeAvatar } from "../../Networking";
+
+export default function AvatarSelectionElement({
+  item,
+  index,
+  username,
+  changeProfileAvatar,
+  baseUrl,
+  currentAvatar,
+}) {
+  let backgroundColour;
+  if (currentAvatar === index) {
+    backgroundColour = "lightgrey";
+  } else {
+    backgroundColour = "white";
+  }
+
```

Comment:
could u just do 
let background colour = "white"
if (currentAvatar === index) {
    backgroundColour = "lightgrey"}
idk might be wrong tho hahah